code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -175,7 +178,7 @@ static void Mumble_Connect()
 
 	_initVoiceChatConfig();
 
-	g_mumbleClient->ConnectAsync(g_netLibrary->GetCurrentPeer(), fmt::sprintf("[%d] %s", g_netLibrary->GetServerNetID(), g_netLibrary->GetPlayerName())).then([](concurrency::task<MumbleConnectionInfo*> task)
+	g_mumbleClient->ConnectAsync(!g_mumble.overrideAddress.empty() ? net::PeerAddress::FromString(fmt::sprintf("%s:%d", g_mumble.overrideAddress, g_mumble.overridePort), g_mumble.overridePort, net::PeerAddress::LookupType::NoResolution).get() : g_netLibrary->GetCurrentPeer(), fmt::sprintf("[%d] %s", g_netLibrary->GetServerNetID(), g_netLibrary->GetPlayerName())).then([](concurrency::task<MumbleConnectionInfo*> task)
```

Comment:
Actually, you can even store an optional<net::PeerAddress> directly, so you can do something like

```cpp
g_mumble.overridePeer ? *g_mumble.overridePeer : g_netLibrary->GetCurrentPeer()
```